api/types/constants.go

```diff
@@ -629,6 +629,8 @@ const (
 	// created from the Okta service.
 	OriginOkta = common.OriginOkta
 
+	OriginSCIM = common.OriginSCIM
```

Comment:
Missing godoc.

Reply:
Also, why do we need to create another constant here instead of just using the ones from the common package directly?

Reply:
Deleted